sign/eddilithium3/eddilithium.go

```diff
@@ -0,0 +1,210 @@
+// eddilithium3 implements the hybrid signature scheme Ed25519-Dilithium3.
+package eddilithium3
+
+import (
+	"crypto"
+	cryptoRand "crypto/rand"
```

Comment:
nit: `cryptorand`

> They are lower case, with no under_scores or mixedCaps

https://blog.golang.org/package-names

Reply:
@armfazh What do you think?  I think it's worth here to make it easy to see this is `crypto`s `rand` sub package.  (We also do it like that in `ed25519` and `ed448`.)

Reply:
Just to be clear, I think the package alias is good, I'm just suggesting `cryptorand` not `cryptoRand`.

Yes, looks like it is done the same way in `ed25519` and `ed448`. I would make the same comment for those occurrences.

Reply:
https://github.com/golang/go/blob/dd150176c3cc49da68c8179f740eadc79404d351/src/crypto/ed25519/ed25519.go#L22

Reply:
they are using an alias for `crypto/rand` because there exists a variable called `rand` in one of the functions.
So renaming this variable allows to use `crypto/rand` without alias.

Reply:
Regardless of collisions, there is an argument for using `cryptorand` alias anyway, to avoid any accidental use of `math/rand` instead of `crypto/rand`.

This always makes me nervous... if you call `rand.Read(...)` then `goimports` could give you the wrong one, and you have to double-check it did the right thing.